test/TestLogger.AcceptanceTests/DotnetTestFixture.cs

```diff
@@ -11,17 +11,19 @@ public static class DotnetTestFixture
     {
         private const string NetcoreVersion = "netcoreapp3.1";
 
-        public static string RootDirectory { get; } = Path.GetFullPath(
+        private static string assembly = string.Empty;
```

Comment:
Since this will introduce state, will this block us from running tests in parallel in future?

Reply:
Fixed